.travis.yml

```diff
@@ -31,10 +31,14 @@ script:
       cd docs;
       make html;
       cd ..;
-      python -m doctr deploy --gh-pages-docs . --key-path deploy_key.enc;
-      python -m doctr deploy --gh-pages-docs docs --key-path deploy_key.enc;
-      python -m doctr deploy --no-require-master --gh-pages-docs "docs-$TRAVIS_BRANCH" --built-docs docs/_build/html --key-path deploy_key.enc;
-      python -m doctr deploy --no-require-master --key-path deploy_key.enc --no-sync --command "echo test";
+      if [[ -z "${DOCTR_DEPLOY_ENCRYPTION_KEY}" ]]; then
+          echo "DOCTR_DEPLOY_ENCRYPTION_KEY not set, likely building a PR from untrusted repository";
```

Comment:
That's only supposed to be called with `canpush` is True. https://github.com/drdoctr/doctr/blob/84d09b19b4f045c93f06516dae78c55662b3db26/doctr/travis.py#L185. So either we have a bug or Travis does (maybe `$TRAVIS_PULL_REQUEST` is not always set properly). 

It would be nice if we could keep this as an error, because it is a legitimate error if someone didn't set up their .travis.yml correctly. Maybe there is a more robust way to detect pull requests than looking at the environment variables. 

Let's open an issue for this.